addon/-private/validated-changeset.ts

```diff
@@ -49,6 +49,12 @@ const AFTER_VALIDATION_EVENT = 'afterValidation';
 const AFTER_ROLLBACK_EVENT = 'afterRollback';
 const defaultValidatorFn = () => true;
 const defaultOptions = { skipValidate: false };
+const isEmberDataObject = (obj: Object) => {
```

Comment:
what do you think about this take to check if an ember-data model? #406

Reply:
I believe it doesn't do the same thing. I could be wrong, just testing out to see if all tests pass.

Reply:
Did you see if that worked?  We had another issue on EmberData objects and the fix on #406 seemed to fix it for them...

Reply:
Also noting that either way is fine with me.  Whatever seems more canonical is what we should do across the repo.

Reply:
@snewcomer I just tried a similar approach and it didn't work. If there isn't any other way to see if an object is and Ember Data object, then I think this change is fine.

Reply:
Ok.  I don't want to drag this PR on as you seem to have fixed quite a good bug that surely will help others!

Ultimately, we would want a `util` to determine if something is an Ember Data object.  If #406 isn't it, we probably need to make sure we have the canonical way of doing so.

@luketheobscure